Client.xcodeproj/project.pbxproj

```diff
@@ -8281,44 +8281,44 @@
 			shellPath = /bin/sh;
 			shellScript = "if [ $ACTION != \\\"indexbuild\\\" ]; then\n   bash $PWD/bin/nimbus-fml.sh -n MozillaAppServices\nfi\n";
 		};
-		C874A4E327F62C5B006F54E5 /* Swiftlint */ = {
```

Comment:
Had a look and turns out the changes are fine (has no impact) - although let's monitor next time as this shouldn't be changed.